### PR TITLE
feat: Add contract for saving table address

### PR DESCRIPTION
**Pull request by howjmay**
This contract is used for saving the address which is mapping from
a certain table name.

### CODE REVIEW

**Comment by jserv** [comment]
Is Access Control List (ACL) taken into consideration? That is, we would track ACL accordingly.

**Comment by howjmay** [comment]
I will implement another contract component for ACL.

File: contract/audit_log.sol [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+contract audit_log{
+    // The table for mapping DB table name and address which save the the audit log of DB
+    // operations.
+    mapping(string => address) auditLogMap;
+
+    /**
+     * @dev Add a new DB table name and corresponding ethereum public key
+     */
+    // FIXME We need to implement a mechanism to avoid tableName conflicting among different organizations
```
Comment (jserv) [inline contract/audit_log.sol:16]: How about introducing a hashtable and/or dictionary to abstract the naming?

Comment (howjmay) [reply contract/audit_log.sol:16]: `mapping` is a hashtable in solidity.  Do you mean that we need another mapping to construct a  layer hashtable structure?

**Pull request merged**